ui/Screen/RegisterUser.svelte

```diff
@@ -0,0 +1,103 @@
+<script>
+  import { ModalLayout, StepCounter } from "../DesignSystem/Component";
+
+  import PickHandleStep from "./RegisterUser/PickHandleStep.svelte";
+  import SubmitRegistrationStep from "./RegisterUser/SubmitRegistrationStep.svelte";
+
+  import { Text, Title } from "../DesignSystem/Primitive";
+  import { gql } from "apollo-boost";
+  import { getClient, mutate } from "svelte-apollo";
+  import { pop } from "svelte-spa-router";
+
+  let step = 1;
+  let handle = "cloudhead"; // TODO(merle): Get actual user profile (id, name, avatar)
+  const id = "1234";
+
+  const nextStep = () => {
+    step += 1;
+  };
+
+  const previousStep = () => {
+    step -= 1;
+  };
+
+  const client = getClient();
+  const REGISTER_USER = gql`
+    mutation($handle: ID!, $id: ID!) {
+      registerUser(handle: $handle, id: $id) {
+        messages {
+          ... on UserRegistration {
+            handle
+            id
+          }
+        }
+      }
+    }
+  `;
+
+  let response;
+  let errorMessage;
+  const registerUser = async () => {
+    console.log("Register handle, id ", handle, id);
+    try {
+      response = await mutate(client, {
+        mutation: REGISTER_USER,
+        variables: {
+          handle: handle,
+          id: id
+        }
+      });
+    } catch (error) {
+      errorMessage = error;
+      console.log("Register user error: ", errorMessage);
+    } finally {
+      console.log("Register user respone", response);
+      pop();
```

Comment:
It's strange to see a stray `pop` without any context, what does it do?

Reply:
`pop` brings you to your last location, in this case your user profile and specifically the tab which was opened before the register modal was opened.

Reply:
Understood, thanks for the explanation.